soc/realtek/ec/rts5912/reg/reg_spim.h

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2025 Realtek, SIBG-SD7
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SOC_REALTEK_RTS5912_REG_SPIM_H
+#define ZEPHYR_SOC_REALTEK_RTS5912_REG_SPIM_H
+
+/**
+ * @brief SPIM Controller (SPIM)
+ */
+
+struct spim_reg {
+	union {
+		volatile uint32_t MSCMDL;
+		struct {
+			volatile uint32_t CMD0: 8;
+			volatile uint32_t CMD1: 8;
+			volatile uint32_t CMD2: 8;
+			volatile uint32_t CMD3: 8;
```

Comment:
None of the other register files in this directory use bitfields for register definitions.  Use access macros instead for consistency.

Accessing the register directly through the bitfields will generate extra register read and writes.

```
struct spi_reg {
    uint32_t CMDL;
    uint32_t CMDH;
...
};

#define CMDL_CMD0_MASK GENMASK(7,0)
```